MapboxVision/Services/Constants.swift

```diff
@@ -8,4 +9,6 @@ enum Constants {
     static let millisecondsInSecond: Double = 1000
     
     static let frameDuration: TimeInterval = 1 / 30.0
+
+    static let attitudeOrient: CMAttitudeReferenceFrame = .xArbitraryZVertical
```

Comment:
Why we choose `xArbitraryZVertical ` as a ref frame?
Did you consider roll and yaw for axes?

Reply:
I suppose 'cause it has the lowest CPU usage and we don't rely on that data